src/status_emitter.rs

```diff
@@ -642,13 +639,9 @@ fn create_error(
             })
             .collect(),
         footer: vec![],
-        opt: FormatOptions {
-            color: colored::control::SHOULD_COLORIZE.should_colorize(),
```

Comment:
colors here

Reply:
From docs (if i understand correctly, https://docs.rs/annotate-snippets/latest/annotate_snippets/struct.Renderer.html#method.plain), `Renderer::plain()` is simple not colored output, while `Renderer::styled()` is some default color scheme.